layers/+lang/latex/packages.el

```diff
@@ -62,6 +62,7 @@
         "C" 'TeX-command-master
         "e" 'LaTeX-environment
         ;; Find a way to rebind tex-fonts
+<<<<<<< a3635045c388daf0f5c3c85f3d01656c0931bb30
```

Comment:
You had a merge conflict, that can be recognized as

```text
<<<<<<< COMMIT HASH
…
=======
…
>>>>>>> COMMIT HASH
```

When you rebase, if your changes can not be applied as-is because there was some conflicting changes upstream, then you add to manually resolve it, then save the file, then add the file `git add FILE`, then continue the rebase `git rebase --continue`.

i.e. you have to manually merge the first block with the second one :-)